app/src/lib/stores/app-store.ts

```diff
@@ -419,6 +431,14 @@ export class AppStore extends TypedBaseStore<IAppState> {
         currentPullRequest: null,
         isLoadingPullRequests: false,
       },
+      compareState: {
+        formState: { kind: ComparisonView.None },
+        commitSHAs: [],
+        aheadBehindCache: new ComparisonCache(),
+        allBranches: [],
```

Comment:
This is being pulled in from `branchesState` when we initialize the compare:

https://github.com/desktop/desktop/blob/63db804fda7bb84716b0781b08aa9fcbeb3a1ac4/app/src/lib/stores/app-store.ts#L732-L751

It's only really for the branches list inside the compare tab (`_executeCompare` doesn't depend on it).

The benefit of this duplication means that we can pass just `ICompareState` into the `CompareSidebar` component, rather than also passing in `IBranchesState`.

I'd be :thumbsup: to revisiting this after shipping and see if we can simplify this area.